style.css

```diff
@@ -0,0 +1,72 @@
+
+  h1 {  color: rgb(167, 101, 221); font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+
+  }
+  body {
+      background: linear-gradient(to bottom, #000000 0%,#17cbeb 100%);
+  }
+  
+  .box {
+          color: rgb(167, 101, 221);
+          width: 200px; 
+          height: 200px; 
+          font-size: 100px;
+          text-align: center; 
+          margin-left: auto;
+          background-color: rgb(12, 7, 1);    
+     
+  }
+  
+  div {
+      border-radius: 100%; 
+      height: 200px; 
+      width: 200px;
+    
+  }
+  .click{
+      color: rgb(167, 101, 221);
+      height: 100px;
+      width: 100px;
+      font-size: medium;
+      background-color: rgb(3, 14, 15);
+      font-style: normal;
+  }
+  
+  .circle1 {
+      background-color: rgb(148, 90, 223); 
+      left: 400px;
+      top: 100px;
+      position: absolute;
+      }
+  .circle2 {
+      background-color: rgb(48, 24, 133); 
+      position: absolute; 
+      left: 475px; 
+      top: 180px;
+   }
+  
+   .circle3 {
+      background-color: #6856ce;
+      position: absolute;
+      left: 550px;
+      top: 280px ;
+    }
+    
+  .circle4 {
+      background-color: rgb(181, 137, 238); 
+      position: absolute;
+      left: 625px;
+      top: 380px; 
+  
+                      }  
+  .circle5 {
+      background-color: rgb(63, 61, 170); 
+      position: absolute; 
+      left: 675px ;
+      top: 480px;
+  
+  }
+  
+      
+  
```

Comment:
Well done here. We will go over later what you can do to select groups of css classes and allow you to position accordingly.

Also, make sure you only have one CSS declaration per line (end each in a semicolon) 

Try to be consistent about spacing.